app/services/pii/attributes.rb

```diff
@@ -1,13 +1,17 @@
 module Pii
   DEPRECATED_PII_ATTRIBUTES = [
     :otp, # https://github.com/18F/identity-idp/pull/1661
+    # Address fields that we might be able to remove. We don't think these were ever used in prod
+    :prev_address1, :prev_address2, :prev_city, :prev_state, :prev_zipcode
   ].freeze
 
   Attributes = RedactedStruct.new(
     :first_name, :middle_name, :last_name,
+    # The user's residential address
     :address1, :address2, :city, :state, :zipcode, :same_address_as_id,
+    # The address on a user's state-issued ID, which may be different from their residential address
+    :state_id_address1, :state_id_address2, :state_id_city, :state_id_state, :state_id_zipcode,
```

Comment:
I know we already discussed this via slack, but I wanted to note that a lot of the codebase uses `state_id_jurisdiction`, rather than `state_id_state`.  For example, [this AAMVA check](https://github.com/18F/identity-idp/blob/main/app/controllers/concerns/idv/verify_info_concern.rb#L9-L13).

Reply:
It also looks like @eileen-nava  is using/expecting state_id_number to be on PII in [PR-7979](https://github.com/18F/identity-idp/pull/7979/files). It does not look like we added this value.  (If is is being added else where please ignore, just calling it out in case not.) Do you think that is something we should consider adding in in the PR?

![Screenshot 2023-03-13 at 3 58 18 PM](https://user-images.githubusercontent.com/125507397/224841457-0074c575-415e-4164-a219-eef2db3b9102.png)

Reply:
Agreed. I'm going to rename `state_id_state` to `state_id_jurisdiction` and add a comment

Reply:
Updated in a6d56cf7c233ce220bf43129e2cd6828026fd9b0